src/app/story/directives/basic.component.ts

```diff
@@ -108,8 +112,94 @@ export class BasicComponent implements OnDestroy, DoCheck {
   ) {
     this.tabSub = tab.model.subscribe((s: StoryModel) => {
       this.story = s;
+      this.series = new SeriesModel(null, s.parent);
       this.loadVersionTemplates();
+
+      concat(
+        this.loadDistribution(),
+        this.loadPodcast(),
+        this.loadUnpublished()
+      ).subscribe({complete: () => {
+          if (this.story.isNew && !this.podcast.complete && this.missingDrafts()) {
+            this.alertForDrafts();
+          }
+        }});
+    });
+  }
+
+  missingDrafts() {
+    return (this.hasDrafts !== null && !this.hasDrafts);
+  }
+
+  alertForDrafts() {
+    const msg = `The more drafts you add, the better we can support your podcast.
+     Please add
+     <a href="/series/${this.series.id}/plan">
+       all of your known upcoming episodes
+     </a>
+     to the Production Calendar.`;
+
+    this.modal.alert('Missing Episode Drafts', msg,  () => {  });
+  }
+
+  loadDistribution() {
+    let obs = this.series.loadRelated('distributions');
+
+    obs.subscribe(() => {
+      this.distribution = this.series.distributions.find((d) => d.kind === 'podcast');
+    });
+
+    return obs;
+  }
+
+  loadPodcast() {
+    if (!this.distribution) {
+      this.podcast = null;
+      return of({});
+    }
+
+    let obs = this.distribution.loadRelated('podcast');
+    obs.subscribe(() => {
+      this.podcast = this.distribution.podcast;
     });
+
+    return obs;
+  }
+
+  loadUnpublished() {
+    this.hasDrafts = null;
+
+    const halParams = {
+      page: 1,
+      per: 1,
+      filters: 'state=unpublished',
+      sorts: 'released_at:desc',
```

Comment:
Hit an edge case - apparently this sort puts null-released-at stories _before_ ones with a time set?

Digging around CMS filters for a way around that - noticed there's an `after` filter that wires into [this scope](https://github.com/PRX/cms.prx.org/blob/master/app/models/story.rb#L141).  Things work as expected if I drop the sort, and just do a:

```js
filters: `after=${new Date().toISOString()}`
```

Which actually might also save the trouble of initing the StoryModel at all.